packages/tpetra/core/src/Tpetra_FECrsMatrix_def.hpp

```diff
@@ -71,7 +71,17 @@ FECrsMatrix(const Teuchos::RCP<const fe_crs_graph_type>& graph,
      ( *graph->activeCrsGraph_!= fe_crs_graph_type::FE_ACTIVE_OWNED,std::runtime_error,
       "Input graph must be in FE_ACTIVE_OWNED mode when this constructor is called.");
 
-  activeCrsMatrix_     = Teuchos::rcp(new FEWhichActive(FE_ACTIVE_OWNED_PLUS_SHARED));
+  bool start_owned = false;
+  if (! params.is_null ()) {
+    if (params->isParameter ("start owned")) {
```

Comment:
Minor, but, since you are willing to set `start_owned` as default (see line below), then you might as well not check whether `start owned` is a parameter or not...